src/Lexer.php

```diff
@@ -352,4 +375,24 @@ public static function decodeJson($json)
     {
         return json_decode($json, true);
     }
+
+    /**
+     * Escape plain text output before mixing in a html context.
+     * 
+     * This should be used on any input or attributes in a delta operation.
+     * For escaping input, use Line->getInput() instead as it keeps track of only doing it once for nested inline listeners.
+     * 
+     * @since 1.2.0
+     * 
```

Comment:
no newline. Also we do not align variables (tab alignment for variable type)

@param string $value
@return string

Maybe the the docs could contain some more infos like

@param string $value The value to encode
@param return The encoded value, as long as $escpaedInput is enabled.